src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -111,6 +112,14 @@ class IntelliJPlugin implements Plugin<Project> {
         def ivyFile = createIvyRepo(project, extension)
         def version = extension.version
 
+        project.configurations.create("provided")
+
+        project.sourceSets.configure {
+            main.compileClasspath += project.configurations.getByName("provided")
+            test.compileClasspath += project.configurations.getByName("provided")
+            test.runtimeClasspath += project.configurations.getByName("provided")
```

Comment:
Is this really needed in `test.runtimeClasspath`? If so, why it's not required for `main.runtimeClasspath`?